packages/rest-api/src/constants/bridgeMap.ts

```diff
@@ -569,6 +569,13 @@ export const BRIDGE_MAP = {
         '0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56',
       ],
     },
+    '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE': {
+      decimals: 18,
+      symbol: 'BNB',
+      origin: ['RFQ.BNB'],
+      destination: [],
```

Comment:
BNB isn't available as a token to choose from on the FE. Think it's related to two things:

(1)
Don't think these destination arrays should be empty, otherwise the route generator won't know what tokens are connected to what. See `RFQ.WLD` for example. The `yarn maps:generate` should pick that up.

(2)
Looking at the actual quote. This isn't something we support (BNB on 56 to Eth on 1). Is this supposed to be wrapped BNB on 1?

```
{
"origin_chain_id": 56,
"origin_token_addr": "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
"dest_chain_id": 1,
"dest_token_addr": "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
"dest_amount": "0",
"max_origin_amount": "0",
"fixed_fee": "2088962468892000",
"relayer_addr": "0xbe75079fd259a82054cAAB2CE007cd0c20b177a8",
"origin_fast_bridge_address": "0x5523D3c98809DdDB82C686E152F5C58B1B0fB59E",
"dest_fast_bridge_address": "0x5523D3c98809DdDB82C686E152F5C58B1B0fB59E",
"updated_at": "2024-10-28T14:36:06Z"
},
```

Reply:
resolved during standup and over TG, relayer meant to quote ETH on BNB Chain